docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2'
```

Comment:
Let's add some info about this docker-compose and the docker-compose-confluent-cloud to the README so we know what it's used for

Reply:
A section on docker-compose files has been added to the README.